### ISSUE TITLE
[A11y]: 'text-muted' on a grey bg violates WCAG color contrast standards

### ISSUE THREAD

**Issue by st0nebraker** [sourcegraph/about#5538]
### Audit type

ARC Toolkit

### User journey audit issue

#5500

### Problem description

`text-muted` text color on a grey background doesn't pass the ARC Toolkit test. Other pages using `text-muted` on a grey bg should be updated along with this.
![image](https://user-images.githubusercontent.com/59381432/176559930-df664c10-d6b2-4d7e-b356-97acb77c14e8.png)

### Expected behavior

The new [Figma DLS](https://www.figma.com/file/o1QRtdQI0ozKq0n7ATrKlx/Marketing-DLS?node-id=3212%3A21096) shows a slightly darker grey HEX for these items.

### Additional details

Our new [Brand Colors from Figma](https://www.figma.com/file/o1QRtdQI0ozKq0n7ATrKlx/Marketing-DLS?node-id=18%3A101) should be reflected on the [Handbooks Brand Guidelines](https://handbook.sourcegraph.com/departments/engineering/design/brand_guidelines/color/) page as well, to eliminate confusion.
